supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -152,23 +161,68 @@ async def create_new_session(
         session_data: Union[None, Dict[str, Any]],
         user_context: Dict[str, Any],
     ) -> SessionContainer:
+        log_debug_message("createNewSession: Started")
+        output_transfer_method = self.config.get_token_transfer_method(
+            request, True, user_context
+        )
+        if output_transfer_method == "any":
+            output_transfer_method = "header"
+
+        log_debug_message(
+            "createNewSession: using transfer method %s", output_transfer_method
+        )
+
+        # TODO: Move TLD checking logic here
+
+        disable_anti_csrf = output_transfer_method == "header"
+
         session = await session_functions.create_new_session(
-            self, user_id, access_token_payload, session_data
+            self,
+            user_id,
+            disable_anti_csrf,
+            access_token_payload,
+            session_data,
         )
+
+        # access_tokens: Dict[TokenTransferMethod, str] = {} # FIXME: type
+
+        session_methods_to_call: List[Callable[[Any], Any]] = []
+
+        transfer_methods: List[TokenTransferMethod] = available_token_transfer_methods  # type: ignore
+        for transfer_method in transfer_methods:
+            request_access_token = get_token(request, "access", transfer_method)
+            if (
+                transfer_method != output_transfer_method
+                and request_access_token is not None
+            ):
+                # access_tokens[transfer_method] = request_access_token
+                session_methods_to_call.append(
+                    # lambda response: clear_session(self.config, response, transfer_method)
+                    partial(
+                        clear_session,
+                        self.config,
+                        transfer_method,
+                    )
+                )
+
         access_token = session["accessToken"]
         refresh_token = session["refreshToken"]
-        id_refresh_token = session["idRefreshToken"]
+
         new_session = Session(
             self,
             access_token["token"],
             session["session"]["handle"],
             session["session"]["userId"],
             session["session"]["userDataInJWT"],
+            output_transfer_method,
         )
         new_session.new_access_token_info = access_token
         new_session.new_refresh_token_info = refresh_token
```

Comment:
you don't need these anymore as you can append to `session_methods_to_call` to add the access, refresh, front token etc.. just like how it's done in the node SDK. So you could create a function called `attach_session_tokens_to_response` and do the same thing that the node SDK does (note that in the node SDK, this function is called `attachCreateOrRefreshSessionResponseToExpressRes` which is a bad name and will be changed).